src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -641,6 +644,7 @@ public void windowClosing(WindowEvent e) {
             bp.updateSearchManager();
             // Set correct enabled state for Back and Forward actions:
             bp.setBackAndForwardEnabledState();
+            bp.getUndoManager().postUndoRedoEvent();
```

Comment:
Sorry, I still don't get it. 
So I close the window, why should the undo manager care about it??